app/rtc_s1.py

```diff
@@ -0,0 +1,1262 @@
+#!/usr/bin/env python
+
+'''
+RTC Workflow
+'''
+
+import os
+import sys
+from subprocess import call
+import time
+
+import isce3
+import logging
+import numpy as np
+from osgeo import gdal
+import h5py
+import argparse
+
+from s1reader.s1_burst_slc import Sentinel1BurstSlc
+
+from rtc.geogrid import snap_coord
+from rtc.runconfig import RunConfig
+from rtc.yaml_argparse import YamlArgparse
+from rtc import mosaic_geobursts
+
+from nisar.workflows.h5_prep import set_get_geo_info
+
+BASE_DS = f'/science/CSAR'
+FREQ_GRID_SUB_PATH = 'RTC/grids/frequencyA'
+FREQ_GRID_DS = f'{BASE_DS}/{FREQ_GRID_SUB_PATH}'
+
+logger = logging.getLogger('rtc_s1')
+
+class Logger(object):
```

Comment:
I assume Logging stuff and specially this class could be easily outside rtc_s1.py?